packages/hash/api/src/model/entityType.model.ts

```diff
@@ -116,11 +120,22 @@ class __EntityType {
 
   static async getEntityType(
     client: DBClient,
-    params: { entityTypeId: string },
+    params: { entityTypeId?: string; entityTypeVersionId?: string },
```

Comment:
We currently have that situation in search-loader. But it should be simple enough to change

Reply:
So it seems that since we are subscribing on live changes from the database, the `entity_versions` table does not contain the `entityTypeId`, which is what we get through the realtime queue, so we'd have to look up the entity for the `entityVersionId` and get the `entityTypeId` from there. 

We are saving a database call for every modification on `entity_versions` if we don't look up the `entityTypeId`, which I would prefer. Thoughts on that?

Reply:
Hmm I see, I haven't considered that. In that case I think it's fine to leave it as is for now then, my instinct is that we'll want to modify how the search-loader receives updates - and not have this happen at the postgres level. But this is out of scope for this PR.